samples/src/main/java/org/opensearch/client/samples/util/CommonUtil.java

```diff
@@ -0,0 +1,46 @@
+package org.opensearch.client.samples.util;
+
+import java.io.IOException;
+import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.client.opensearch.OpenSearchClient;
+import org.opensearch.client.opensearch._types.mapping.IntegerNumberProperty;
+import org.opensearch.client.opensearch._types.mapping.Property;
+import org.opensearch.client.opensearch._types.mapping.TypeMapping;
+import org.opensearch.client.opensearch._types.query_dsl.Query;
+import org.opensearch.client.opensearch.core.SearchRequest;
+import org.opensearch.client.opensearch.core.SearchResponse;
+import org.opensearch.client.opensearch.indices.CreateIndexRequest;
+import org.opensearch.client.opensearch.indices.IndexSettings;
+
+public class CommonUtil {
```

Comment:
Please remove this class, as per my understanding, the `samples` are self-sufficient code snippets people could copy/paste (from the document fe), such utils classes make it unusable.

Reply:
That makes sense, I'll revert my changes